delivery_recipe/Delivery/steering_group_meetings.md

```diff
@@ -0,0 +1,27 @@
+# Steering Group Meetings
+
+## What is a steering group?
+
+A steering group is generally made up of senior managers within the client’s business. Steering groups are set up to provide direction to a project, programme of work or a constant organisational change.
```

Comment:
'Projects' are referred to a lot. That's okay sometimes, but let's emphasise work streams etc.